marimo/_cli/export/commands.py

```diff
@@ -133,18 +133,37 @@ async def start() -> None:
         "If not provided, the HTML will be printed to stdout."
     ),
 )
+@click.option(
+    "--sandbox",
+    is_flag=True,
+    default=False,
+    show_default=True,
+    type=bool,
+    help=(
+        "Run the command in an isolated virtual environment using "
+        "`uv run --isolated`. Requires `uv`."
+    ),
+)
 @click.argument("name", required=True, callback=validators.is_file_path)
 @click.argument("args", nargs=-1, type=click.UNPROCESSED)
 def html(
     name: str,
     include_code: bool,
     output: str,
     watch: bool,
+    sandbox: bool,
     args: tuple[str],
 ) -> None:
-    """
-    Run a notebook and export it as an HTML file.
-    """
+    """Run a notebook and export it as an HTML file."""
+    import sys
+
+    from marimo._cli.sandbox import prompt_run_in_sandbox
```

Comment:
this is awesome!

would it be possible to
1) add this to `def ipynb` if `--include-outputs` is `True` (because otherwise you don't need them installed)
2) add tests for both. you can just tests, something like:

```pyython
    @pytest.mark.skipif(not HAS_UV, reason="uv is required for sandbox tests")
    def test_cli_export_html_sandbox(temp_marimo_file: str) -> None:
        p = subprocess.run(
            [
                "marimo",
                "export",
                "html",
                temp_marimo_file,
                "--sandbox",
            ],
            capture_output=True,
        )
        assert p.returncode == 0, p.stderr.decode()
        html = normalize_index_html(p.stdout.decode())
        # Remove folder path
        dirname = path.dirname(temp_marimo_file)
        html = html.replace(dirname, "path")
        assert '<marimo-code hidden=""></marimo-code>' not in HTML
```

Reply:
Both done. I hope the tests are correct.

Reply:
@metaboulie looks amazing!